composer.json

```diff
@@ -25,9 +25,11 @@
         }
     ],
     "require": {
-        "php": ">=7.0",
+        "php": ">=7.1",
         "ext-json": "*",
-        "socialconnect/common": "^1.1"
+        "psr/http-client": "^1.0",
+        "psr/simple-cache": "^1.0",
+        "guzzlehttp/psr7": "^1.6"
```

Comment:
Any particular reason you chose this lib instead of `zend-diactoros`? :slightly_smiling_face:. 

`guzzlehttp/psr7` v1.6 is PHP 5.4+ and doesn't have PSR-18 support while [zend-diactoros](https://github.com/zendframework/zend-diactoros) v2 is PHP 7.1+ and has PSR-18 support.

Reply:
Related to PSR-18/17/7 i moved forward with 

https://github.com/SocialConnect/auth/pull/84/commits/1ed1152731c346d176238c6acf4880e25f7cb654
https://github.com/SocialConnect/auth/pull/84/commits/a05f97fb172ebbde9602ee4f94706a8d77482b9f

In near time i am planing to move `Http\Client` outside project (guzzlehttp/psr7 will not be require in socialconnect)

And user will be allowed to use any http client with any http stack, but there is a problem

Prototyping:

https://github.com/SocialConnect/auth/pull/84/commits/35973e6bb8b7570d78fee18dd8faea84899a1dd0

![image](https://user-images.githubusercontent.com/572096/60688854-72e48880-9ec0-11e9-847d-750ca186396a.png)

For example I am using

`Http\Client\Curl` as PSR-18 (http client)
and `RequestFactory`/`StreamFactory` from `Zend\Diactoros` with they own implementation

Reply:
Great idea, I'll follow the updates :slightly_smiling_face:.